### PR TITLE
Cas sentryguns pricedown and dinamic one for omnisentryguns

### PR DESCRIPTION
**Pull request by cuberound**
# About the pull request

Lowers price of normal sentryguns, they just are not worth 500 points each. Lowers price of launchbay to 200 from 400, why have it so expensive when there is no reason to ever buy second one. And for the major change, makes omnisentryguns price scale with how many you bought, they start at 300 and go up 100 each time you buy one, so lets do some math. First one is for 300, easy. If you buy 5 they are 500 points each, nice. When you buy 10 they are 800 each aka the same as now, and if you try to buy more they will be hella more expensive. This is here to help those who think this will cause sentry spam, it is more against it, then what we have now.

# Explain why it's good for the game

This gives marines another tool to use, most pilot will buy at least 1 omni sentry with this PR, most will not buy more then 3 or 5. 
500 for alamo sentry is just noobtrap basicly, pilots just feel like they are griefing cas when they buy them, and they basically have no impact, if you are agains 3 sentryguns inside alamo, balance that, do not just make it expensive. 

Basically, tool for marines to use in game = good.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:

balance: transport sentrygun 500 -> 200
balance: launchbay 400 -> 200
balance: omnisentrygun 800->300, but they go up 100 in price each time you buy one

/:cl:

### CODE REVIEW

File: code/modules/cm_marines/vehicle_part_fabricator.dm [hunk 1]
```diff
@@ -29,6 +32,7 @@
 /obj/structure/machinery/part_fabricator/dropship/ui_data(mob/user)
 	return list(
 		"points" = get_point_store()
```
Comment (fira) [inline code/modules/cm_marines/vehicle_part_fabricator.dm:34]: missing ,

did you test this?

Comment (cuberound) [reply code/modules/cm_marines/vehicle_part_fabricator.dm:34]: I did some untested changes on it recently, trying to finde out how UI works. it is draft for a reason

**Comment by ihatethisengine** [comment]
Why not make Alamo sentries go up in price too? And yeah, huge price is one way to balance 3 sentries inside Alamo, and you just made it a lot easier without any downsides. Also they are pretty strong, saying they have no impact is just wrong.

**Comment by cuberound** [comment]
> Why not make Alamo sentries go up in price too? And yeah, huge price is one way to balance 3 sentries inside Alamo, and you just made it a lot easier without any downsides. Also they are pretty strong, saying they have no impact is just wrong. the alamo sentryguns are only strong when they are all 3 inside the alamo and xenos can attack only from south, sentryguns outside just... die after 3 shots

**Comment by ULTRANUT99** [comment]
> Why not make Alamo sentries go up in price too? And yeah, huge price is one way to balance 3 sentries inside Alamo, and you just made it a lot easier without any downsides. Also they are pretty strong, saying they have no impact is just wrong.



> Why not make Alamo sentries go up in price too? And yeah, huge price is one way to balance 3 sentries inside Alamo, and you just made it a lot easier without any downsides. Also they are pretty strong, saying they have no impact is just wrong.

1. you thumbsed up the CAS UI change which is a massive nerf to CAS, breaks CAS literally (it crashes and bricks constantly), and indicates you've never played PO once 
2. by the time xenos get to Alamo they've already won. Alamo sentries just let more marines evac FOB, which means more players in round=more fun evac hold=more fun

**Comment by Zonespace27** [comment]
Thank you for the PR, but I am currently not interested in the majority of the changes here.

I would be alright with a lowering of the ground sentries price by a bit (100-200).
